### ISSUE TITLE
Add more info about posts like categories, tags, author etc. 

### ISSUE THREAD

**Issue by IonTulbure** [10up/distributor#1188]
### Is your enhancement related to a problem? Please describe.

Hi. Currently post info in distributor_page_pull table is pretty limited (page: ?page=pull&status=new page ). 
Would be great to have more columns with info like post author, tags, categories, excerpt, time of day (currently it is displayed on hover)

### Designs

![distributor-enhancement](https://github.com/10up/distributor/assets/42011629/e409a130-5131-4009-bdb2-660084ee26b0)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jeffpaul** [comment]
@IonTulbure you should be able to extend that page to add additional filters/sorts by leveraging the [existing hooks in the codebase](https://10up.github.io/distributor/)

**Comment by IonTulbure** [comment]
@jeffpaul Yeah, i already managed to get most info. I also needed to exclude Published posts from the Pulled page. $item param from docs offers info only from the source site. 

But if i want to get the new post id and afterwards post status ? This is were i couldn't do it with filters or actions. I had to use $wpdb to get what i need.

**Comment by cousett** [comment]
Not a developer so I am plussing this without having to extend myself.
